index.d.ts

```diff
@@ -269,17 +269,41 @@ declare module 'plaid' {
     amount: number | null;
   }
 
+  interface ACHNumbers {
+    account: string,
```

Comment:
nit: can we use `;` as the trailing delimiter on the typings (as we do elsewhere in the file)?